src/modules/datasets/hooks.js

```diff
@@ -13,3 +17,15 @@ export const useDatasetDataTypes = () => {
   }, [dispatch, datasetsByID]);
   return useAppSelector((state) => state.datasetDataTypes);
 };
+
+export const useDatasetDataTypeByID = (datasetId) => {
```

Comment:
should be plural I think - multiple data types?

Reply:
Oh true